dcrtimed/params.go

```diff
@@ -30,7 +29,7 @@ type params struct {
 // to emulate the full reference implementation RPC API.
 var mainNetParams = params{
 	Params:              chaincfg.MainNetParams(),
-	WalletRPCServerPort: netparams.MainNetParams.GRPCServerPort,
+	WalletRPCServerPort: "9111",
```

Comment:
Please don't remove actual proper constants.

Reply:
This is not a "proper" constant -- the netparams package was never intended for external consumption.  It is no longer available in the latest module either.

Reply:
I really hate that we don't have proper conts for ports. Why wouldn't that be exported?

Reply:
a little copying would be better here. I dare say you probably shouldn't want to import a package like this in the first place, because that in turn also pulls in some version of chaincfg, which may not be the chaincfg major version your application is using (in this case, that's exactly what was happening).

Reply:
I absolutely hate that we don't have constants for constant things.